styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,117 @@
 .footer{
-  
-}
\ No newline at end of file
+  color:#fff;
+  ul{
+    list-style: none;
+    padding: 0px;
+    margin:0px;
+    li{
+      padding: 0px;
+      margin:5px 0px;
+      a{
+        color:#aaa;
+        font-size: 1.2rem;
+      }
+      span.menu-title{
+        font-size: 1.3rem;
+      }
+    }
+  }
+}
+/*  Mobile   */
+@media only screen and (min-width:50px) and (max-width: 767px)  {
+  .fluid-footer-container{
+    padding: 0px 0px;
+  }
+  .footer{
+    width: 100%;
+    padding: 0px 0px;
+    ul{
+      li{
+        padding: 0px;
+        margin:0px 0px;
+        text-align: center;
+        display: block;
+        width: 100%;
+        span.menu-title{
+          border-bottom:1px solid #2E3238;
+          background-color:#23262b;
+          padding:12px;
+          width:100%;
+          display: block;
+          font-size: 1.3rem;
+          &:hover{
+            cursor: pointer;
+            cursor: hand;
+          }
+        }
+        ul{
+          li{
+            ul{
+              li{
+                padding:0px;
+                border-bottom: 1px solid #3b4047;
+                a{
```

Comment:
This tree of selectors is out of hand and demonstrates why classes are the superior selector, not just in performance, but in authoring.

`.footer ul li ul li ul li a` is the final selector. In addition to being less-than-desireable CSS, it reflects what is probably hairy markup.

`.footer-menu-link` or something to that effect would be much cleaner, performant, and maintainable.

Are we using something already available in Bootstrap, or are we making it all from scratch?